docs/security.rst

```diff
@@ -0,0 +1,66 @@
+Security analysis
+====================
+
+You can use the module ``pypowsybl.security`` in order to perform security analysis on a network.
+
+AC security analysis
+-----------------------
+
+To perform a security analysis, you need at least a network and a contingency on this network.
+In the result there are violations detected with the initial loadflow on the network.
+These violations are collected in pre_contingency_result. The results contain also
+the violations created by the contingency, they are collected by contingency in post_contingency_results :
+
+.. doctest::
+     :options: +NORMALIZE_WHITESPACE
+    >>> import pypowsybl as pp
```

Comment:
sphinx is touchy, you need to add a blank line here:
for now the code does not show up in the generated HTML :) 

Same thing in the code section for monitored elements